### DISCUSSION TITLE
My application was rejected with unfair reason

### DISCUSSION THREAD

**Discussion by OlenaKolyada** [community/community#141982] [GitHub Education]
### Hi everyone,

I don't know who will be reading this. I applied for student free program but my application was rejected without any good reason. I'm studying at AFPA - official institution in France that fit all the criteria for this licence. However I was rejected. Not even possible to make an appel. Do you guys really think you are doing good here?

**Comment by sandeepmaddheshiya** [comment]
In my opinion, it’s crucial for organizations to provide clear reasons for application decisions and to allow applicants the opportunity to appeal or seek clarification. Transparency in the process fosters trust and encourages applicants to continue engaging with the program.
